WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3067,12 +3109,31 @@ public void onTrackableEvent(TrackableEvent event, Map<String, String> propertie
     }
 
     @Override public void onStoryComposerLoadRequested(ArrayList<Object> mediaFiles, String blockId) {
+        if (mLoadStoryFromStoriesPrefsUseCase.anyMediaIdsInGutenbergStoryBlockAreCorrupt(mediaFiles)) {
+            // unfortunately the medaiIds seem corrupt so, show a dialog and bail
```

Comment:
```suggestion
            // unfortunately the mediaIds seem corrupt so, show a dialog and bail.
```

Reply:
this piece is going away in https://github.com/wordpress-mobile/WordPress-Android/pull/13103 (see commit b8d6c4f7ca81bfc2c55025f48173c70ee3434d0f) - thanks for the suggestion!